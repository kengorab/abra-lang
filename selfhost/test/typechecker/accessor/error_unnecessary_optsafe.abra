val s = "abc"
s?.length