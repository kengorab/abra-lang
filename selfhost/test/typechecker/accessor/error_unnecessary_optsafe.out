Error at %FILE_NAME%:2:2
Unnecessary use of '?.' operator
  |  s?.length
      ^
(The lhs value's type here is known to be a non-Option type)
